client/tx/tx.go

```diff
@@ -170,14 +179,6 @@ func SignWithPrivKey(
 	return sigV2, nil
 }
 
-func checkMultipleSigners(mode signing.SignMode, tx authsigning.Tx) error {
```

Comment:
as per comment above, let's keep this function. It needs an `isLastSigner bool` argument though.